Assets/LeapMotion/Core/Scripts/Utils/Infix.cs

```diff
@@ -73,13 +73,18 @@ public static class Infix {
     /// Infix sugar for Vector3.SignedAngle(a, b).
     /// </summary>
     public static float SignedAngle(this Vector3 a, Vector3 b, Vector3 axis) {
+#if UNITY_2017_1_OR_NEWER
       return Vector3.SignedAngle(a, b, axis);
+#else
+      Debug.LogError("SignedAngle not supported in Unity versions before 2017.1");
```

Comment:
We should provide a custom implementation of SignedAngle instead of just logging an error, since our code may come to rely on it as well.